README.md

```diff
@@ -4,7 +4,7 @@
 [![Module LTS Adopted'](https://img.shields.io/badge/Module%20LTS-Adopted-brightgreen.svg?style=flat)](http://github.com/CloudNativeJS/ModuleLTS)
 [![IBM Support](https://img.shields.io/badge/IBM%20Support-Frameworks-brightgreen.svg?style=flat)](http://ibm.biz/node-support)
 
-**This module is in Active LTS mode, new features are no longer accepted.**
+**This module is in Maintenance LTS mode, new features are no longer accepted, and only receives critical bug fixes.**
```

Comment:
This is not true, LB3 stays in Active LTS until Dec 2019 - see https://strongloop.com/strongblog/lb3-extended-lts/


Version | Status | Published | Active LTS Start | Maintenance LTS Start | End-of-life
-- | -- | -- | -- | -- | --
LoopBack 4 | Current | Oct 2018 | – | – | Apr 2021(minimum)
LoopBack 3 | Active LTS | Dec 2016 | Oct 2018 | Dec 2019 (revised) | Dec 2020 (revised)
LoopBack 2 | Maintenance LTS | Jul 2014 | Dec 2016 | Oct 2018 | Apr 2019

The same comment applies to other relevant changes below too.

Reply:
you're right. thanks for pointing that out.